src/patterns/dosdp-patterns/left_teeth.yaml

```diff
@@ -0,0 +1,91 @@
+pattern_name: right__left_teeth
+pattern_iri: http://purl.obolibrary.org/obo/uberon/patterns/right_left_teeth.yaml
+
+description: "This pattern defines primary and secondary teeth according to laterality."
+
+contributors:
+  - https://orcid.org/0000-0001-9625-1899 # Bill Duncan
+
+classes:
+  thing: "owl:Thing"
+  Homo sapiens: "NCBITaxon:9606"
+  calcareous tooth: "UBERON:0001091"
+  jaw region: "UBERON:0011595"
+  maxilla: "UBERON:0002397"
+  mandible: "UBERON:0001684"
+
+relations:
+  in lateral side of: "BSPO:0000126"
+  in right side of: "BSPO:0000121"
+  in left side of: "BSPO:0000120"
+
+annotationProperties:
+  hasExactSynonym: "oboInOwl:hasExactSynonym"
+  present_in_taxon: "RO:0002175"
+  contributor: "dc:contributor"
+  creation_date: "oboInOwl:creation_date"
+  hasDbXref: "oboInOwl:hasDbXref"
+  hasSynonymType: "oboInOwl:hasSynonymType"
+
+vars:
+  parent_class: "calcareous tooth"       # UBERON:0001091
+  jaw_region: "jaw region"               # UBERON:0011595
+  FMA_label: "xsd:string"                # e.g., right upper third secondary molar tooth
+  human_preferred: "UBERON:core#HUMAN_PREFERRED"
+
+data_vars:
+  laterality: "xsd:string"     # e.g., right, left
+
+data_list_vars:                # separate multiple entries by "|"
+  synonym: "xsd:string"        # e.g., ADA tooth 1, FDI tooth 18
+  creator: "xsd:string"        # e.g., https://orcid.org/0000-0001-9625-1899
```

Comment:
The type creator is not string because it's an ORCID.

Reply:
Thank @anitacaron. I changed the range of `creator` to `xsd:anyURI`.